bower.json

```diff
@@ -14,5 +14,20 @@
     "angular-ui-router": "~0.2.18",
     "bootstrap": "~3.3.6",
     "owasp-password-strength-test": "~1.3.0"
+  },
+  "overrides": {
+    "bootstrap": {
+      "main": [
+        "dist/css/bootstrap.css",
+        "dist/css/bootstrap-theme.css",
+        "less/bootstrap.less"
+      ]
+    },
+    "jquery": {
+      "main": []
+    }
+  },
+  "resolutions": {
+    "angular": "~1.3"
```

Comment:
@mleanos it should say what dependency cannot resolve the correct version of Angular. Let's fix that since that is the root cause of the issue rather than just resolving it.